src/main/java/net/mcreator/generator/setup/folders/CurrentFolderStructure.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.generator.setup.folders;
+
+import net.mcreator.generator.Generator;
+import net.mcreator.generator.GeneratorConfiguration;
+import net.mcreator.generator.GeneratorFlavor;
+import net.mcreator.generator.GeneratorUtils;
+import net.mcreator.ui.workspace.resources.TextureType;
+import net.mcreator.workspace.Workspace;
+
+import javax.annotation.Nullable;
+import java.io.File;
+
+/**
+ * This version of the folder structure estimator uses folder structure of the latest supported Forge generator available.
+ * <p>
+ * If the provided workspace has generator defined, it will use that generator for the folder structure.
+ */
+class CurrentFolderStructure extends AbstractFolderStructure {

```

Comment:
Maybe call this e.g. `FolderStructure1194` to avoid renaming this class later down the road (the other one being `FolderStructurePre1193` then)?

Reply:
So this one would later be copied, for instance, as `FolderStructurePre120`?